meta-mbl-distro/scripts/create-update-payload

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+
+"""
+This script will create a bootloader update payload tar archive.
```

Comment:
What about rootfs updates and app updates?

I'd be happy for you to say that they're not within the scope of this PR/task, but I don't think we should limit this comment to bootloaders.

Reply:
I think rootfs and app updates need some thought and are probably out of scope for this. I'll edit the docstring so it's not limited to bootloaders.